managing-cf/usage-events-and-billing.html.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Usage Events and Billing
+---
+
+## Usage Events
+Usage events can be used by Cloud Foundry operators to construct billing information for apps and service instances.
+
+Despite being similar in name, usage events are different from regular Cloud Foundry events (`/vX/events`). Cloud Foundry events should NOT be used for billing purposes; They are recorded regardless of the action succeeding and are not guaranteed to be in the correct order.
```

Comment:
Change to say, "...usage events are different from Cloud Foundry audit events (`/v2/events`).  The audit events stored in the `events` table should not be used for billing purposes...